src/impl_methods.rs

```diff
@@ -59,6 +59,11 @@ impl<A, S, D> ArrayBase<S, D> where S: Data<Elem=A>, D: Dimension
         self.dim.size()
     }
 
+    /// Return whether the array has any elements
+    pub fn is_empty(&self) -> bool {
```

Comment:
Thanks for adding this and tests.

Sorry to nitpick on such a small PR -- but `len` & `len_of` must be next to each other in the docs. So this method should move down a step.

Reply:
Oh no problem! That makes sense.